qemu/tests/cfg/transfer_file_over_ipv6.cfg

```diff
@@ -1,10 +1,13 @@
 - bigfile_transfer_over_ipv6:
     virt_test_type = qemu
-    only Linux
     type = transfer_file_over_ipv6
-    vms += " vm2"
+    vms = "vm1 vm2"
     image_snapshot = yes
     filesize = 4096
     file_trans_timeout = 2400
     file_md5_check_timeout = 600
     dd_cmd = "dd if=/dev/zero of=%s oflag=direct bs=1M count=%d"
+    Linux:
+        tmp_dir = "/tmp/"
```

Comment:
Confirm with @yanan-fu , RHEL 8.0 guest make `/tmp` as a seperate partition, '3.4G' is the total size now, please use `/var/tmp` as your temp dir. Thanks.